pkg/kubelet/config/file_linux.go

```diff
@@ -79,6 +80,10 @@ func (s *sourceFile) watch() error {
 }
 
 func (s *sourceFile) processEvent(e *inotify.Event) error {
+	// Ignore file start with dots
+	if strings.HasPrefix(e.Name, ".") {
```

Comment:
ack, and do you think we need to doc this behaviour in description of `--pod-manifest-path`?